src/c++/perf_analyzer/custom_load_manager.h

```diff
@@ -95,17 +96,19 @@ class CustomLoadManager : public RequestRateManager {
   /// \return cb::Error object indicating success or failure.
   cb::Error GetCustomRequestRate(double* request_rate);
 
- private:
+ protected:
```

Comment:
Why make this protected? If it's for testing purposes, we already use the friend paradigm to give access to private members.

Reply:
back to private